src/App.js

```diff
@@ -1,108 +1,101 @@
+} from "grommet";
+import { FormClose, Notification } from "grommet-icons";
 
 const theme = {
   global: {
     colors: {
-      brand: '#228BE6',
+      brand: "#228BE6",
     },
     font: {
-      family: 'Roboto',
-      size: '14px',
-      height: '20px',
+      family: "Roboto",
+      size: "14px",
+      height: "20px",
     },
   },
 };
 
 const AppBar = (props) => (
-  <Box
-    tag='header'
-    direction='row'
-    align='center'
-    justify='between'
-    background='brand'
-    pad={{ left: 'medium', right: 'small', vertical: 'small' }}
-    elevation='medium'
-    style={{ zIndex: '1' }}
+  <Header
+    background="brand"
+    pad={{ left: "medium", right: "small", vertical: "small" }}
+    elevation="medium"
     {...props}
   />
 );
 
-class App extends Component {
-  state = {
-    showSidebar: false,
-  }
-  render() {
-    const { showSidebar } = this.state;
-    return (
-      <Grommet theme={theme} full>
-        <ResponsiveContext.Consumer>
-          {size => (
-            <Box fill>
-              <AppBar>
-                <Heading level='3' margin='none'>My App</Heading>
-                <Button
-                  icon={<Notification />}
-                  onClick={() => this.setState({ showSidebar: !this.state.showSidebar })}
-                />
-              </AppBar>
-              <Box direction='row' flex overflow={{ horizontal: 'hidden' }}>
-                <Box flex align='center' justify='center'>
-                  app body
-                </Box>
-                {(!showSidebar || size !== 'small') ? (
-                  <Collapsible direction="horizontal" open={showSidebar}>
-                    <Box
-                      flex
-                      width='medium'
-                      background='light-2'
-                      elevation='small'
-                      align='center'
-                      justify='center'
-                    >
-                      sidebar
-                    </Box>
-                  </Collapsible>
-                ): (
-                  <Layer>
-                    <Box
-                      background='light-2'
-                      tag='header'
-                      justify='end'
-                      align='center'
-                      direction='row'
-                    >
-                      <Button
-                        icon={<FormClose />}
-                        onClick={() => this.setState({ showSidebar: false })}
-                      />
-                    </Box>
-                    <Box
-                      fill
-                      background='light-2'
-                      align='center'
-                      justify='center'
-                    >
-                      sidebar
-                    </Box>
-                  </Layer>
-                )}
+const App = () => {
+  const [showSidebar, setShowSidebar] = useState(false);
+  return (
+    <Grommet theme={theme} full>
+      <ResponsiveContext.Consumer>
```

Comment:
I feel like usually we do the following then use `size` to set different rendering in our story book should we change that to match here?
` const size = useContext(ResponsiveContext);`

Im okay with leaving as is though just my thoughts

Reply:
Changed to use `const size = useContext(ResponsiveContext);` in latest commit